### PR TITLE
feat: created reusable component for WACI Share credential preview

### LINKED ISSUE #1390: Refactor credential banner in WACI share
- [ ] Refactor credential preview/details code into a reusable component
- [ ] Add vault container
- [ ] Show credential details by default when only 1 credentials is being shared
image

### PR DESCRIPTION
**Pull request by birtony**
Fixes #1390

image
image
image

Signed-off-by: Anton Biriukov <anton.biriukov@securekey.com>

### CODE REVIEW

**Review by rolsonquadras — APPROVED** [review]

**Comment by sudeshrshetty** [comment]
@birtony leave vault name empty if you are not getting it from backend. "Default Vault" is actual vault name

File: cmd/wallet-web/src/pages/WACIShare.vue [hunk 1]
```diff
@@ -282,7 +238,9 @@ export default {
         credentials.map((credential) => {
           const manifest = this.getManifest(credential);
           const processedCredential = this.getCredentialDisplayData(credential, manifest);
-          this.processedCredentials.push({ ...processedCredential, showDetails: false });
+          // TODO: add logic to retrieve the Vault name in which the credential is stored
+          const vaultName = 'Default Vault';
```
[the remark this answers is not in the document]
Comment (sudeshrshetty) [reply cmd/wallet-web/src/pages/WACIShare.vue:242]: as we decided earlier, leave issue numbers in TODO comments

[the remark this answers is not in the document]
Comment (birtony) [reply cmd/wallet-web/src/pages/WACIShare.vue:242]: Filed #1410 @sudeshrshetty

[the remark this answers is not in the document]
Comment (birtony) [reply cmd/wallet-web/src/pages/WACIShare.vue]: image

**Review by sudeshrshetty — APPROVED** [review]

**Pull request merged**